services/models.py

```diff
@@ -22,8 +22,8 @@ def __str__(self):
 
 
 class ServiceConnection(models.Model):
-    profile = models.ForeignKey(Profile, on_delete=models.PROTECT)
-    service = models.ForeignKey(Service, on_delete=models.PROTECT)
+    profile = models.ForeignKey(Profile, on_delete=models.CASCADE)
+    service = models.ForeignKey(Service, on_delete=models.CASCADE)
```

Comment:
would it make sense to `PROTECT` here, to not allow deleting a service object if it has connected profiles? something @klempine might weigh in on

Reply:
profile fk would be `CASCADE`, yes.

Reply:
Yeah. `PROTECT` on `Service`, `CASCADE` on `Profile` but there should be something in the future to protect profiles connected to certain services (at least in certain state).